docs/source/networks.rst

```diff
@@ -0,0 +1,311 @@
+be used to check if a network is full.
+
+Multiple networks
+^^^^^^^^^^^^^^^^^
+
+In experiments configured for a number of `practice_repeats` or
+`experiment_repeats` higher than one, the `create_network` method is called
+multiple times, once for every repeat. This means that an experiment can have
+multiple networks at the same time.
+
+The experiment setup code assigns each network a role of `practice` or
+`experiment`, depending on how it was created. The experiment class allows
+experiment developers to query networks by role (practice, experiment), or by
+state (full, not full). For example:
+
+::
+
+    all_networks = exp.networks()
+    full_networks = exp.networks(full=True)
+    not_full_networks = exp.networks(full=False)
+    practice_networks = exp.networks(role='practice')
+    full_experiment_networks = exp.networks(role='experiment', full=True)
+
+Generally, the networks created at experiment setup will all be of the same
+type, but there's nothing to stop an imaginative experimenter from creating a
+different network type based on some condition, thus having multiple networks
+of different types.
+
+Common networks in Dallinger
+----------------------------
+
+Many experiments will be able to just use one of Dallinger's existing
+networks, rather than defining their own. Lets look at the basic networks
+that can be used out of the box.
+
+Empty
+^^^^^
+
+There are experiments where participants do not need to interact with each
+other at all. Generally, in this case, a source will be required. The empty
+network does not connect any nodes with each other, which results in a
+series of isolated nodes. The only exception is, if a source node is added,
+it will be connected to all existing nodes, which means that it's possible to
+send a stimulus to all network nodes, regardless of their isolation.
+
+Chain
+^^^^^
+
+A chain network, also known as `line` network, connects each new node to the
+previous one, so that nodes can receive information from their parent, but
+cannot send information back. In other words, it's a one way transmission
+chain. In general, it's useful to have a source as the first node, so that
+an initial experiment stimulus is transmitted to the each node through the
+chain. Note that this network explicitly prohibits a source to be added after
+any node, so the source has to come first.
+
+This network can be useful for experiments where some piece of information,
+for example, a text, needs to be modified or interpreted by each participant
+in succession.
+
+Star
+^^^^
+
+A start network uses its first node as a central node, and nodes created
+after that have a bidirectional connection (`both`) with that node. This
+means the central node can send and receive information from/to all nodes,
+but every other node in the network can only communicate with the central
+node.
+
+A source can't be used as a first node, since the connections to it need to
+be in both directions.
```

Comment:
Can we add an example use case here? Maybe:

"This network can be useful for experiments where one user has a supervisory role over others who are working individually, for example making a decision based on advice from the other players"